tests/conftest.py

```diff
@@ -57,7 +57,8 @@ def service_url(environment):
     if environment == "prod":
         base_url = "https://api.service.nhs.uk"
     else:
-        base_url = f"https://{environment}.api.service.nhs.uk"
+        http_base = "https://"
```

Comment:
Should this new variable be extracted so it can be used for both this else case as well as the if block above?

Reply:
The prod case is a fixed string, so doesn't need to have any variable substitution.
I'm hpaay to leave it as it is